test_cases/reverse_coarse.json

```diff
@@ -69,10 +69,9 @@
           {
             "layer": "locality",
             "locality": "San Juan",
-            "county": "Municipio de San Juan",
+            "county": "San Juan",
             "region": "Puerto Rico",
-            "dependency": "Puerto Rico",
-            "country": "United States"
```

Comment:
removing country here and not replacing it with anything else, like empire, feels a little unsettling

Reply:
Yeah agreed. @trescube please opine.